generation/new_client_hermetic_build/README.md

```diff
@@ -0,0 +1,160 @@
+
+Therefore, the "proto path" value we supply to the command is `google/cloud/alloydb`.
+
+We will publish a single module for a service that includes all versions in this path. Once the service has been published once, any future additional versions will automatically be generated via OwlBot.
+
+### Name pretty (`name_pretty`)
+
+The corresponding value in the Cloud Drop page is `title`.
+
+Example: `AlloyDB API`
+
+### Product Docs (`product_docs`)
+
+The corresponding value in the Cloud Drop page is `documentation_uri`.
+The value must starts with "https://".
+
+Example: `https://cloud.google.com/alloydb/docs`
+
+### REST Docs (`rest_docs`)
+
+The corresponding value in the Cloud Drop page is `rest_reference_documentation_uri`.
+The value must starts with "https://".
+
+Example: `https://cloud.google.com/alloydb/docs/reference/rest`
+
+If they exist, add them as a flag to the python command below like:
+`--rest-docs="https://cloud.google.com/alloydb/docs/reference/rest" \`
+
+### RPC Docs (`rpc_docs`)
+
+The corresponding value in the Cloud Drop page is `proto_reference_documentation_uri`.
+The value must starts with "https://".
+
+Example: `https://cloud.google.com/speech-to-text/docs/reference/rpc`
+
+If they exist, add them as a flag to the python command below like:
+`--rpc-docs="https://cloud.google.com/speech-to-text/docs/reference/rpc" \`
+
+### API description (`api_description`)
+
+The corresponding value in the Cloud Drop page is `documentation.summary` or `documentation.overview`.
+If both of those fields are missing, take the description from the product page above. Use the first sentence to keep it concise.
+
+Example:
+``` 
+    AlloyDB for PostgreSQL is an open source-compatible database service that
+    provides a powerful option for migrating, modernizing, or building
+    commercial-grade applications.
+ ```
+
+### Transport (`transport`)
+
+This variable represents the type of requests the library will make to
+its corresponding service. It can be `grpc` (default), `http` or `both`. In
+practice, it is mainly used to create a `.repo-metadata.json` file.
+
+### Destination Name (`destination_name`)
+
+This variable represents the folder name to be created in the root of the
+monorepo. For example, if it is `java-example-library`, then a new folder
+with the generated library will be created in
+`google-cloud-java/java-example/library`.
+
+### Distribution Name (`distribution_name`)
+
+This variable determines the Maven coordinates of the generated library. It
+defaults to `com.google.cloud:google-cloud-{api_shortname}`. This mainly affect
+the values in the generated `pom.xml` files.
+
+## Advanced Options
+
+In case the steps above don't show you how to specify the desired options, you can
+run the `new-client.py` script in your local evironment. The advanced options
+not shown in the section above **cannot be specified in the Github Action**,
```

Comment:
Does this mean Github action doesn't allow optional parameters?

Reply:
Yes, there is a limit of 10 parameters for github actions. See https://github.com/orgs/community/discussions/8774